### DISCUSSION TITLE
No app for apple iPad

### DISCUSSION THREAD

**Discussion by ReeceEvans1701** [olivierfriard/BORIS#329] [Q&A]
Just wondering if there is any plan to bring an app for BORIS to the apple App Store? I’m a student using an iPad but I’m going to have to buy a cheap computer in order to use BORIS for my course

**Comment by olivierfriard** [comment]
Hi,

In theory the BORIS App for Android should also run on iPad because it is
developed using the Kivy framework (see https://kivy.org).
So test it and let me know.
No chance to have it on the App store because an Apple developer account
costs 100$/Y ;-)

If you want a very cheap computer you can choose the Raspberry Pi 4 with
Ubuntu (~50$) with an old TV.

o


On Thu, Oct 8, 2020 at 12:04 PM ReeceEvans1701 <notifications@github.com>
wrote:

> Just wondering if there is any plan to bring an app for BORIS to the apple
> App Store? I’m a student using an iPad but I’m going to have to buy a cheap
> computer in order to use BORIS for my course
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/olivierfriard/BORIS/issues/329>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABR2AYSWLE4HYAGJRHP3OFTSJWFCXANCNFSM4SIREYXQ>
> .
>


-- 
University of Torino
Life Sciences and Systems Biology department
Via dell'Accademia Albertina, 13 10123 TORINO (Italy)
tel: +39 011 6704542
http://penelope.unito.it/friard/
http://orcid.org/0000-0002-0374-9872

**Comment by ReeceEvans1701** [comment]
hi,

i just tried to download the android one but because i cannot access google play on my ipad it won’t let me, thanks for letting me know about that cheap option for the laptop that’s a massive help, really appreciate that thank you for trying to help me

reece

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by olivierfriard** [comment]
>
> i just tried to download the android one but because i cannot access
> google play on my ipad it won’t let me,


BORIS is not on the Google Play store.



> thanks for letting me know about that cheap option for the laptop that’s a
> massive help, really appreciate that thank you for trying to help me
>
>
The Raspberry Pi is not a laptop:
https://www.raspberrypi.org/products/raspberry-pi-4-model-b/
You must have a monitor (an old TV should be OK), a USB keyboard and an USB
mouse.

o

**Comment by ReeceEvans1701** [comment]
oh right i already have an old usb keyboard and mouse so i should be able to use that, and i think the launcher id have to use is on google play

reece

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
